carto/exceptions.py

```diff
@@ -20,15 +20,38 @@ class CartoException(Exception):
 
 
 class CartoRateLimitException(CartoException):
+    """
+    This exception is raised when a request is rate limited by SQL or Maps APIs (429 Too Many Requests HTTP error).
+    More info about CARTO rate limits: https://carto.com/developers/fundamentals/limits/#rate-limits
+
+    It extends CartoException with the rate limits info, so that any client can manage
+    when to retry a rate limited request.
+    """
     def __init__(self, response):
+        """
+        Init method
+
+        :param response: The response rate limited by CARTO APIs
+        :type response: requests.models.Response class
+
+        :return:
+        """
         super().__init__(response.text)
         self.limit = int(response.headers['Carto-Rate-Limit-Limit'])
         self.remaining = int(response.headers['Carto-Rate-Limit-Remaining'])
-        self.retryAfter = int(response.headers['Retry-After'])
+        self.retry_after = int(response.headers['Retry-After'])
         self.reset = int(response.headers['Carto-Rate-Limit-Reset'])
 
     @staticmethod
-    def isResponseRateLimited(response):
+    def isRateLimited(response):
```

Comment:
forgot to mention this method should not be camelCase 😛

Reply:
Looks like we could include [a plugin](https://github.com/PyCQA/pep8-naming) to detect this kind of things

Reply:
:ouch: I can't see them!